pkg/svcat/service-catalog/class_test.go

```diff
@@ -207,4 +207,38 @@ var _ = Describe("Class", func() {
 			Expect(actions[0].(testing.GetActionImpl).Name).To(Equal(fakeClassName))
 		})
 	})
+	Describe("CreateClassByName", func() {
+		It("Calls the generated v1beta1 create method with the passed in class", func() {
+			realClient := &fake.Clientset{}
+			realClient.AddReactor("create", "clusterserviceclasses", func(action testing.Action) (bool, runtime.Object, error) {
```

Comment:
I don't believe that you need to define a Create Reactor here? Pretty sure that the default test fake handles this for you already. Does it still work if you remove lines 213-215?